ui/main/src/app/business/services/modal.service.ts

```diff
@@ -52,19 +52,16 @@ export class ModalService {
     }
 
     public static openConfirmationModal(title: string | I18n, message: string | I18n): Promise<boolean> {
-        const titleTranslated = this.getTranslatedValue(title);
-        const messageTranslated = this.getTranslatedValue(message);
-
         const modalConfig: ModalConfig = {
-            title: titleTranslated,
-            message: messageTranslated,
+            title: title,
+            message: message,
             buttons: [
                 {
-                    label: this.translationService.getTranslation('button.cancel'),
+                    label: new I18n('button.cancel'),
                     id: 'cancel'
                 },
                 {
-                    label: this.translationService.getTranslation('button.ok'),
+                    label: new I18n('button.ok'),
                     id: 'ok',
                     isSelected: true
```

Comment:
Could you do the same for the other methods (using I18n)
And remove all translation feature from the class (and getTranslatedValue method)

Reply:
Done.